app/views/devise/sessions/new.html.erb

```diff
@@ -1,3 +1,30 @@
+<<<<<<< HEAD
```

Comment:
Have you testet the app after merging main back into this branch? The conflict markers are still in here. This will break.

Reply:
Soma was testing when I left. But I can review everything again tomorrow, as early as possible